### PR TITLE
German translation

### PR DESCRIPTION
**Pull request by Sekisback**
The following folders are translated to German by hand, according to the given Keywords from the Cura German translation:

blackmagic
cooling
dual
resolution
about

Mistyping fix in TranslationButton.qml

working on the rest

### CODE REVIEW

**Comment by Ghostkeeper** [comment]
Great work. Thanks for the contributions! I can help out a bit with the syntax mistakes that the tests found, if necessary.

**Comment by Sekisback** [comment]
Hey morning, please feel free to change the syntax if necessary. I'll leave some words, like raft, skirt, brim, bowden tube..., as is., according to the cura settings translation.

I dont know why the ctest failed, if you know what i have to do to avoid this, let me know. Ill update after each complete translated folder

**Comment by Ghostkeeper** [comment]
I've seen that project and was informed of the work by Sehertest via e-mail. That translation was created with a machine translation service though. It was, at least back then, full of the typical sort of mistakes that a computer makes. I understand there has been some effort from the community to improve this. Once it is rid of those mistakes I would accept it. However as it stands, in my [contribution guidelines](https://github.com/Ghostkeeper/SettingsGuide/blob/master/contributing.md#translating) I've explicitly forbidden machine translations (as far as I can detect it; I don't speak every language either). It's not on par with the standards I hold for the Settings Guide, or Cura as a whole for that matter.

I'm going to review these translations now, and will try fixing the formatting mistakes that the tests are complaining about.

File: resources/translations/de_DE/infill/infill_wipe_dist.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+====
+Bei dieser Einstellung bewegt sich die Düse ein wenig weiter als das Ende der Fülllinie. Ziel dieses kleinen Fahrwegs ist es, das Material auf der daneben liegenden Wand abzuwischen. Dadurch verbindet sich die Fülllinie besser mit der Wand.
+
+![Eine Visualisierung der Füllungsüberlappung und des Wischabstands](../images/infill_overlap.svg)
+
+Dieser Fahrweg macht das Objekt zwar stärker, hat aber den großen Nachteil, dass er durch die Wand geführt wird, was eine sichtbare Spur auf der Außenseite des Drucks hinterlässt. Dadurch scheint das Füllmuster stärker durch die Außenhaut hindurch.
+
+Dies gilt nur für die Enden der Fülllinien. Wenn Sie zusammenhängende Fülllinien verwenden, gibt es viel weniger Fülllinienenden.
+
+Übersetzt mit www.DeepL.com/Translator (kostenlose Version)
```
Comment (Ghostkeeper) [inline resources/translations/de_DE/infill/infill_wipe_dist.md:11]: Is it? Please do go through the translations then. Although I think you did make corrections, at least in this article.

File: resources/translations/de_DE/meshfix/remove_empty_first_layers.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+Normalerweise sollte diese Einstellung immer aktiviert sein, um zu verhindern, dass Drucke fehlschlagen, weil die Unterseite nicht ganz glatt ist oder nicht richtig an der Bauplatte ausgerichtet ist. Sie können diese Einstellung jedoch auch deaktivieren, wenn Sie einen Druckauftrag mit mehreren Stufen durchführen, der in der Luft beginnen muss.
+
+Wenn [Stützstrukturen generieren aktiviert ist](../support/support_enable.md), reicht die Unterstützung bis zur Bauplatte, so dass die ersten Schichten nicht mehr leer sind. Es werden keine Schichten entfernt und der Druck bewegt sich nicht nach unten.
+
+Wenn die Einstellung [Slicing-Toleranz](../experimental/slicing_tolerance.md) auf " Exklusiv" gesetzt ist, wird die erste Schicht immer leer sein. Wenn diese Einstellung nicht aktiviert ist, ist die erste Schicht leer, und der Druck schlägt höchstwahrscheinlich fehl.
```
Comment (Ghostkeeper) [inline resources/translations/de_DE/meshfix/remove_empty_first_layers.md:13]:
```suggestion
Wenn die Einstellung [Slicing-Toleranz](../experimental/slicing_tolerance.md) auf "Exklusiv" gesetzt ist, wird die erste Schicht immer leer sein. Wenn diese Einstellung nicht aktiviert ist, ist die erste Schicht leer, und der Druck schlägt höchstwahrscheinlich fehl.
```

File: resources/translations/de_DE/read.me [hunk 3]
```diff
@@ -0,0 +1,9 @@
+complete translated folders:
```
Comment (Ghostkeeper) [inline resources/translations/de_DE/read.me:1]: It's fine to keep track like this, but best not commit those files. You can add them to your .gitignore maybe?

File: resources/translations/de_DE/resolution/infill_line_width.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+"camera_position": [35, 92, 122],
+"settings": {"infill_line_width": 1},
+"layer": 111,
+"colours": 64
+}-->
+![Die Fülllinien sind deutlich breiter als der Rest](../images/infill_line_width.png)
+
+Wenn Sie die Fülllinien breiter machen, kann Ihr Druck stabiler werden und auch die Druckzeit verkürzen. Eine zu starke Vergrößerung kann jedoch zu großen Extrusionsschwankungen führen. Dies führt zu einer Unterextrusion beim Druck der Füllung und zu einer Überextrusion beim Druck dessen, was nach der Füllung kommt, da sich der Fluss durch die Düse nicht schnell genug anpassen kann.
+
+**Die tatsächlichen Fülllinien können breiter ausfallen als in dieser Einstellung angegeben, wenn Sie die Einstellung [Füllschichtdicke] (../infill/infill_sparse_thickness.md) angepasst haben.**
```
Comment (Ghostkeeper) [inline resources/translations/de_DE/resolution/infill_line_width.md:17]:
```suggestion
**Die tatsächlichen Fülllinien können breiter ausfallen als in dieser Einstellung angegeben, wenn Sie die Einstellung [Füllschichtdicke](../infill/infill_sparse_thickness.md) angepasst haben.**
```

File: resources/translations/de_DE/resolution/support_bottom_line_width.md [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+"camera_lookat": [0, 0, 3],
+"settings": {
+    "support_enable": true,
+    "support_bottom_enable": true,
+    "support_bottom_line_width": 0.8
+},
+"layer": 65,
+"colours": 64
+}-->
+![Der Boden der Stützstruktur ( dunkelblau) ist mit breiteren Linien gedruckt als der Rest der Stützstruktur.](../images/support_bottom_line_width.png)
```
Comment (Ghostkeeper) [inline resources/translations/de_DE/resolution/support_bottom_line_width.md:23]:
```suggestion
![Der Boden der Stützstruktur (dunkelblau) ist mit breiteren Linien gedruckt als der Rest der Stützstruktur](../images/support_bottom_line_width.png)
```

File: resources/translations/de_DE/travel/retraction_combing.md [hunk 6]
```diff
@@ -0,0 +1,46 @@
+Combing-Modus
+====
+Beim Combing wird vermieden, die Wände des Drucks zu überqueren, wenn man von einer Stelle zur anderen fährt. Das Überqueren der Wände hinterlässt in der Regel eine Narbe auf der Oberfläche, wenn die heiße Düse aus dem Druck austritt oder eintritt, was im Allgemeinen unerwünscht ist.
```
Comment (Ghostkeeper) [inline resources/translations/de_DE/travel/retraction_combing.md:3]: Correct me if I'm wrong here, but I thought the word "man" could only really apply to people, not 3D printers? That could be a cultural thing though. We have a similar word "men" in Dutch which wouldn't be used when it's about a machine, similar to the English word "one": "when one travels from one place to another".

File: resources/translations/de_DE/travel/retraction_extrusion_window.md [hunk 7]
```diff
@@ -0,0 +1,11 @@
+Fenster „Minimaler Extrusionsabstand“
+====
+Damit das Filament richtig bewegt werden kann, muss das Feederrad in das Filament greifen. Das Einziehen des Materials führt häufig dazu, dass das Feederrad das Filament so weit abnutzt, dass es das Filament nicht mehr richtig greifen kann. Diese Einstellung begrenzt die Anzahl der Einzüge innerhalb einer bestimmten Länge des Filaments, um diese Abnutzung zu verhindern.
+
+Die Einziehvorgänge werden durch die Kombination aus [Maximale Anzahl von Einzügen] (retraction_count_max.md) und dieser Einstellung begrenzt. Das Material darf innerhalb einer durch diese Einstellung des Fensters für die minimale Extrusionslänge angegebenen Länge des Filaments nicht weiter eingezogen werden als der Wert der maximalen Einzugsanzahl.
```
Comment (Ghostkeeper) [inline resources/translations/de_DE/travel/retraction_extrusion_window.md:5]:
```suggestion
Die Einziehvorgänge werden durch die Kombination aus [Maximale Anzahl von Einzügen](retraction_count_max.md) und dieser Einstellung begrenzt. Das Material darf innerhalb einer durch diese Einstellung des Fensters für die minimale Extrusionslänge angegebenen Länge des Filaments nicht weiter eingezogen werden als der Wert der maximalen Einzugsanzahl.
```

**Review by Ghostkeeper — CHANGES_REQUESTED** [review]
I can't say much about the translations themselves, since I'm not a native speaker. I'm not seeing many obvious mistakes. Looks very good!

I've just made a few corrections for typos here and there. And there is one major change that needs to happen before I can accept these translations: The PNG images can't be duplicated. I'll explain why not in the review.

File: resources/translations/de_DE/read.me (hunk 3 above)
Comment (Sekisback) [reply resources/translations/de_DE/read.me:1]: yeap added to gitignore

File: resources/translations/de_DE/travel/retraction_combing.md (hunk 6 above)
Comment (Sekisback) [reply resources/translations/de_DE/travel/retraction_combing.md:3]: you are right...
better 
Beim Combing wird vermieden, die Wände des Drucks zu überqueren, wenn der Drucker von einer Stelle zur anderen fährt.

File: resources/translations/de_DE/images/brim_width.svg [hunk 8]
```diff
@@ -0,0 +1,34 @@
...
+	<path d="M801,370 l50,30 l-10,-20 h80 l-10,20 l50,-30 l-50,-30 l10,20 h-80 l10,-20 z" />
+
+	<!-- Legend. -->
+	<rect x="1000" y="200" width="150" height="150" fill="midnightblue" />
+	<rect x="1000" y="450" width="150" height="150" fill="deepskyblue" />
+	<path d="M1000,775 l50,30 l-10,-20 h80 l-10,20 l50,-30 l-50,-30 l10,20 h-80 l10,-20 z" />
+	<g font-family="sans-serif" font-size="150px">
+		<text x="1200" y="330">Model</text>
+		<text x="1200" y="580">Brim</text>
+		<text x="1200" y="830">Brim width</text>
```
[the remark this answers is not in the document]
Comment (Sekisback) [reply resources/translations/de_DE/images/brim_width.svg:32]: Nop, removed the images folder

**Comment by Sekisback** [comment]
> You should talked with this guy and ask him to add the file to the Github and you can offer also to validate the translation (on my point of view they are many error and typo in these files).

hey, i checked the "translation" also the "corrected" ones. you are absolutely right, there are a lot of mistakes in there. i tried to use the same words that are already used in Cura (fdmprinter.def.json.po) to increase the recognition value.  for me it would be more costly to correct the correction instead of translating it from scratch. I don't see any progress in the translation since the beginning of the year, although it must have taken a lot of time.
Whenever I have time I will continue to translate and upload the individual articles, whether they are then used I would leave up to you.

File: resources/translations/de_DE/top_bottom/ironing_pattern.md [hunk 9]
```diff
@@ -0,0 +1,48 @@
+Ironing Pattern
```
Comment (Ghostkeeper) [inline resources/translations/de_DE/top_bottom/ironing_pattern.md]: Oops, seems like you added a few English files here by accident.

**Comment by Ghostkeeper** [comment]
Let me express again my gratitude for this great work! And for processing my review so quickly. This way I can still get it into the release that is supposed to support Cura 4.12. There's not a lot of ways I can thank you (and 5axes) for the amount of work you put into this. But I can at least review it thoroughly and help with the technical and administrative things. And give you proper attribution.

There's a few leftover things, but those are things I can fix up myself, like the broken links to images and other things the automated tests are complaining about.

> whether they are then used I would leave up to you.

I'm planning to use the current state of the translations for the release of Settings Guide 2.8.0. I hope this doesn't stop you from continuing translating if you wish! The guide is made to handle partial translations because it's difficult for people to make a complete translation within the timespan between two releases of Cura and the Settings Guide.
I'll try to push my fixes of the links downstream to you to reduce the merge conflicts you'd inevitably get.

**Review by Ghostkeeper — APPROVED** [review]

**Comment by Sekisback** [comment]
This is may way tho say thank you for all the work you have done. I use a lot of tools you wrote for cura and with this i want to give something back.
If you have articles for the new features ready. i can translate them first for cura 4.12. 
Sure ill finish the work i have started ;-)

**Pull request merged**